File: examples/verify_sms/4_send_custom_sms_with_custom_sender_id.py
from __future__ import print_function
from telesignenterprise.verify import VerifyClient

customer_id = "customer_id"
secret_key = "secret_key"

phone_number = "phone_number"
my_sender_id = "my_sender_id"  # Client Services must white list any custom sender_id for it to take effect

verify = VerifyClient(customer_id, secret_key)
response = verify.sms(phone_number, sender_id=my_sender_id)

File: examples/verify_sms/4_send_custom_sms_with_custom_sender_id.rb
require 'telesignenterprise'

customer_id = 'customer_id'
secret_key = 'secret_key'

phone_number = 'phone_number'
my_sender_id = "my_sender_id"  # Client Services must white list any custom sender_id for it to take effect

verify_client = TelesignEnterprise::VerifyClient.new(customer_id, secret_key)
response = verify_client.sms(phone_number, sender_id: sender_id)
